backend/src/schema/resolvers/reports.js

```diff
@@ -87,6 +81,60 @@ export default {
           break
       }
 
+      return response
+    },
+  },
+  Query: {
+    reports: async (_parent, _params, { driver, _req, _user }, _resolveInfo) => {
```

Comment:
if we are not using `req` and `user` there is no reason to destructure them and then put an `_`
this is a good practice for `_parent` cause it is the first parameter and we are not using it, but we don't have to destructure anything more than driver, if we are not using it

Reply:
Yes, you are right. @mattwr18 
Done …